client/reader/components/app-title/index.scss

```
@@ -0,0 +1,14 @@
+.app-title {
+	width: min-content;
+	text-decoration: none;
+	padding-inline: 0;
+	height: fit-content;
+
+	&__heading {
```

```suggestion
	.app-title__heading {
```

As this is easier to search if we use full selector as CSS.

---

I think it's a personal preference, since there are benefits to using the parent selector, but I removed it here. 

[327d574](https://github.com/Automattic/wp-calypso/pull/107781/commits/327d57421dfe30bcbd6a8f4f9f841a9596f256ed)